tests/test_data.py

```diff
@@ -1534,131 +1534,6 @@ async def assert_expected_count(expected_report: dict[str, int]):
     )  # 1 NullResource resource and 1 Dummy resource in model v2
 
 
-async def test_resources_report(init_dataclasses_and_load_schema):
```

Comment:
What's the motivation for deleting this?

Reply:
I deleted `get_resources_report` which was only being used by the v1 endpoint

Reply:
Oh I see, this was a test for the helper method.